dev/TOC.md

```diff
@@ -14,6 +14,7 @@
     - [TPC-C Performance Test - v3.0 vs. v2.1](/dev/benchmark/tpcc.md)
     - [DM 1.0-alpha Benchmark Report](/dev/benchmark/dm-v1-alpha.md)
     - [DM 1.0-GA Benchmark Report](/dev/benchmark/dm-v1.0-ga.md)
+    - [TiDB in Kubernetes Benchmark Report](/dev/benchmark/sysbench-in-k8s.md)
```

Comment:
Please make it clearer and specific. See [the Chinese version](https://pingcap.com/docs-cn/v3.0/benchmark/sysbench-in-k8s/).